### PR TITLE
LYD's homework 

### PR DESCRIPTION
**Pull request by LYD-cubing**
All files  are in the file folder named  LYD-cubing under "_2021-finalproject";

### CODE REVIEW

**Comment by LYD-cubing** [comment]
I need review require.

**Comment by LYD-cubing** [comment]
@MuZhou233

File: _2021-finalproject/LYD-cubing/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+## shell命令 
+   认清了shell，命令，终端，Linux系统之间的关系，对权限和管道命令有了新的认识
+eg:
+``` echo 3 | sudo tee brightness -> sudo echo 3 |sudo tee brightness ``` 
+## vim编辑器
+   1.学习了vim的一些常规操作和实用操作，对vim的便利性有了新的认识 
+   比如 “动词” +  “名词” 的命令，可视化块等
+   2.修改并观察 ~/.vimrc,~/.bashrc等配置文件，了解了配置文件的工作
```
Comment (MuZhou233) [inline _2021-finalproject/LYD-cubing/README.md:8]: There is a typo on this line, have you check rich display on github?

**Review by MuZhou233 — APPROVED** [review]

**Comment by LYD-cubing** [comment]
@MuZhou233

**Review by MuZhou233 — APPROVED** [review]
Tell me if you want to submit your project

**Comment by LYD-cubing** [comment]
I want to submit my project.Thank you!  @MuZhou233

**Pull request merged**